src/generator.ts

```diff
@@ -72,7 +72,7 @@ export function generate(node: t.Node | t.PropTypeNode[], options: GenerateOptio
 
 		return filteredNodes
 			.map((prop) => generate(prop, options))
-			.reduce((prev, curr) => `${prev}\n${curr}`);
+			.reduce((prev, curr) => `${prev}\n${curr}`, '');
```

Comment:
This is isn't needed if the PRs is merged. It prevents a crash if no used props were found. Not sure if we should throw or warn but it should be something other than "TypeError: called reduce on empty array without initial value".

Reply:
I agree that it should be handled better than what happens right now, but you can revert it for this PR.

Reply:
done

Reply:
Fixed in https://github.com/merceyz/typescript-to-proptypes/commit/b69112e1011f089b6d5cb60f88ce75b6394252be